proc_scripts/fitting.py

```diff
@@ -1,22 +1,25 @@
 from pyspecdata import *
 from sympy import symbols
-def _fitcurve_initial(s,f_range,direct,indirect,guess):
+import sympy as sp
+import numpy as np
+import logging
+def _fitcurve_initial(s,direct):
     if not s.get_ft_prop(direct):
         raise ValueError("Your data should be in the frequency domain!")
-    curve = s[direct:f_range].sum(direct).real
+    curve = s.real
     return fitdata(curve)
 def _fitcurve_final(f,whichrate,guess):
-    logger.info(strm("Functional form", f.functional_form))
-    logger.info(strm("Functional form", f.functional_form))
+    logging.info(strm("Functional form", f.functional_form))
+    logging.info(strm("Functional form", f.functional_form))
```

Comment:
I'm a bit confused why there is a frequency range here that gets deleted -- what's up w/ that?